src/_locales/en_US/messages.json

```diff
@@ -423,6 +423,20 @@
         "message": "Please add a valid domain or URL",
         "description": "Error text for the form on the Disabled Sites options page tab"
     },
+    "options_disable_tip_title": {
+        "message": "How to disable Privacy Badger without leaving the site",
+        "description": "Title of tip at the top of the Disabled Sites options page tab"
+    },
+    "options_disable_tip_alt": {
+        "message": "First, click on Privacy Badger's icon in your browser toolbar. Then, click the \"$DISABLE_FOR_SITE$\" button.",
```

Comment:
Probably for later ... but this text may be good and useful for everybody, not just accessibility tool users.